app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/settingsFormHandler.ts

```diff
@@ -61,7 +61,12 @@ const hydrateForm = (userPreferences: UserPreferences) => (form: HTMLFormElement
   form.addEventListener('submit', submitHandler)
 }
 
+const hideUnsupportedBrowserContent = () => {
+  document.body.className = `${document.body.className} cookie-settings__body--browser-supported`
```

Comment:
Would it be better to `+=`?

Reply:
Ha - maybe. I've followed https://eslint.org/docs/rules/prefer-template in the past so would generally avoid concatenation and += generally. Like you say it's a matter of style.